common/sgx/tlsverifier.c

```diff
@@ -172,3 +202,136 @@ oe_result_t oe_verify_attestation_certificate(
     oe_free(report);
     return result;
 }
+
+/**
+ * oe_verify_attestation_certificate_with_evidence
```

Comment:
This is a non-SGX specific function, but is in an SGX-only file (unlike oe_get_attestation_certificate_with_evidence which is already in a non-SGX specific file).  Move this outside the sgx directory for symmetry.

Reply:
Moved it under common (non-sgx specific)